litgpt/finetune/adapter.py

```diff
@@ -317,8 +318,9 @@ def fit(
 
 
 @torch.no_grad()
-def validate(fabric: L.Fabric, model: GPT, val_dataloader: DataLoader, eval: EvalArgs) -> torch.Tensor:
-    fabric.print("Validating ...")
+def validate(fabric: L.Fabric, model: GPT, val_dataloader: DataLoader, eval: EvalArgs, verbose: bool = True) -> torch.Tensor:
+    if verbose:
+        fabric.print("Validating ...")
```

Comment:
As for me this print statement doesn't justify `verbose` flag.
Maybe it's better to print this info above, in if-else statement?

Reply:
If it were just in this place, I'd agree. But we also use the `validate` function throughout the training loop.

Reply:
Hard to see the whole code from a smartphone :)
Ok, it makes sense.